mbuild/compound.py

```diff
@@ -2283,16 +2283,16 @@ def to_parmed(self, box=None, title='', residues=None, show_ports=False,
                 atomic_number = None
                 name = ''.join(char for char in atom.name if not char.isdigit())
                 try:
-                    atomic_number = AtomicNum[atom.name]
+                    atomic_number = AtomicNum[atom.name.lower().capitalize()]
```

Comment:
is the `.lower()` needed?

```python3
Python 3.6.7 | packaged by conda-forge | (default, Feb 28 2019, 09:07:38) 
[GCC 7.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> si.capitaliz
KeyboardInterrupt
>>> 'si'.capitalize()
'Si'
>>> 'Si'.capitalize()
'Si'
>>> 'SI'.capitalize()
'Si'
```

Reply:
Nope its not, you are right. I misread the documentation for `capitalize`, fixing now.

Reply:
(I didn't even know that function existed)